<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="menuWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>331</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn1">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>容器管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn2">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>镜像管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn3">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>仓库</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn4">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>镜像导出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
